pkg/render/compliance.go

```diff
@@ -859,7 +859,7 @@ func (c *complianceComponent) complianceBenchmarkerDaemonSet() *appsv1.DaemonSet
 		{Name: "var-lib-kubelet", MountPath: "/var/lib/kubelet", ReadOnly: true},
 		{Name: "etc-systemd", MountPath: "/etc/systemd", ReadOnly: true},
 		{Name: "etc-kubernetes", MountPath: "/etc/kubernetes", ReadOnly: true},
-		{Name: "usr-bin", MountPath: "/usr/bin", ReadOnly: true},
+		{Name: "usr-bin", MountPath: "/usr/local_bin", ReadOnly: true},
```

Comment:
nit: Can you rename this to `/host/usr/bin` or something like that, assuming that it's possible to put this in such a path and not break a bunch of stuff. `local_bin` feels too close to `/usr/local/bin` for my tastes.